main.go

```diff
@@ -78,7 +84,11 @@ func run(cmd *cobra.Command, args []string) error {
 				// fetch latest release
 				chartVer, err := idx.Get(release.Chart.Metadata.Name, "")
 				if err != nil {
-					return err
+					chartVer, err = idx.Get(release.Chart.Metadata.Name, ">0.0.0-0")
+					if err != nil {
+						fmt.Println(err)
+						continue
```

Comment:
so if I'm reading this right:

1. Do a semver constraint check against the "default" constraints (IIRC "0.0.0")
2. If that fails, do a looser semver check (">0.0.0-0")
3. If that fails, display but don't error out.

So I have a few questions in regards to the design:

- why can't we consolidate the two constrain checks to one?
- can we somehow make the semver check constraint configurable through a feature flag?
- I'm a little concerned on the fmt + continue case. It's possible for an error to occur and the system to return a successful exit code where we probably want to fail.

Reply:
1. I assume that users want to default to released versions.  
2. That's possible, but is it really necessary as the current setup just breaks and does not work in these cases.
3. It depends how you define a failure, whatup did not error out just one of the installed charts does not exist in  the repos.  Is that actually a error?